modules/dev-tools/scripts/test.sh

```diff
@@ -15,11 +15,7 @@ usage() {
 }
 
 run_test_script() {
-  # get Chromium executable path
```

Comment:
Also had to additionally break browser testing support as there is no counterpart to `yarn global` in newer versions